detection2landmark:
  ros__parameters:
    camera_frame: "camF_optical_frame"
    detections_topic: "camF/detections"
    family: "36h11"       # tag family
    rotation_weight: 1e-1
    translation_weight: 1e1